{% if page is sameas 'print' %}
{% set h = 3 %}
{% else %}
{% set h = 1 %}
{% endif %}

<div class="page-header"><h{{h+1}}>Summary</h{{h+1}}></div>
Gibson Isothermal Assembly is a method of assembling multiple DNA sequences during a 1 hour incubation in a single reaction volume [<a href="http://www.synbio.org.uk/gibson/resources/Gibson2009_nmeth.1318.pdf">Gibson et al. 2009</a>]. 

<h{{h+3}}>Benefits</h{{h+3}}>
<ul>
	<li>No forbidden sequences - DNA sequences can be joined without regard to restriction sites.</li>
	<li>Complete control of product sequence (scarless).</li>
	<li>&gt;10 DNA sequences can be assembled in a single 1 hour reaction.</li>
</ul>

<h{{h+3}}>Disadvantages</h{{h+3}}>
<ul>
	<li>Cost and delivery time of custom primers.</li>
	<li>Rearrangements are possible between regions of homology close to the termini of substrate sequences (at low frequency).</li>
	<li>Secondary structure can reduce assembly efficiency.</li>
</ul>

<h{{h+2}}>The scope of this manual</h{{h+2}}>

This manual is intended primarily for users with graduate-level knowledge of molecular biology. It introduces users to the principles of Gibson Isothermal Assembly and its application, as well as providing guidance on implementing an efficient workflow and on troubleshooting. The focus is on primer design and the assembly reaction, but also includes descriptions of other steps in a typical Gibson Isothermal Assembly in order to giving the reader an overview of a complete workflow.


<h{{h+1}}>The Gibson Assembly Reaction</h{{h+1}}>
<p>Complementary termini of ~40 bp sequence homology determine the order in which substrate sequences are assembled. The reaction is performed during a 1 hour incubation at 50 &deg;C and proceeds by i) forming 3’ overhangs, ii) annealing complementary termini, iii) gap-filling and iv) nick-sealing:</p>

<p><strong>Key reaction components:</strong></p>

	<img src="./img/summary_0.png" alt="Two fragments of DNA with homologous regions"><br><br>
<ol class="summary_list">
	<li>T5 exonuclease excises nucleotides from the 5&prime; end of dsDNA molecules, exposing 3&prime; overhangs. (Similar reactions will occur at all DNA termini, but for the sake of simplicity we show one assembly junction only).
	<img src="./img/summary_1.png" alt="T5 exonuclease chews back from the 5&prime; end"></li>
	<li>Complementary  3&prime; overhangs anneal.
	<img src="./img/summary_2.png" alt="Complementary  3&prime; overhangs anneal."></li>
	<li>T5 exonuclease (unstable at 50&deg;C) denatures, exposing free 3&prime; overhangs  to Phusion DNA polymerase.
	<img src="./img/summary_3.png" alt="T5 exonuclease denatures."></li>
	<li>Phusion DNA polymerase fills the 3&prime; overhang.
	<img src="./img/summary_4.png" alt="Phusion DNA polymerase fills the 3&prime; overhang."></li>
 	<li>The DNA polymerase releases the DNA once the overhangs are filled, allowing Taq DNA ligase to seals nicks in the phosphodiester backbone (i.e. catalyse the formation of phosphodiester bonds between 5&prime; phosphate and 3&prime; hydroxyl groups).
	<img src="./img/summary_5.png" alt="Taq DNA ligase seals nicks."></li>
</ol>

<p>More than 10 sequences can be assembled in a single reaction, resulting in linear or circular DNA as desired.</p>
<img src="./img/summary_multiple_1.png">

<p>Substrate DNA sequences can be produced easily by PCR.  The assembly reaction can be used directly to transform E. coli cells, without prior purification. DNA assembly using Gibson Isothermal Assembly is therefore flexible, quick and inexpensive.</p>


<p><strong>Refs</strong></p>
<p>Gibson, Daniel G, Lei Young, Ray-Yuan Chuang, J Craig Venter, Clyde A Hutchison, and Hamilton O Smith. “Enzymatic Assembly of DNA Molecules up to Several Hundred Kilobases.” <em>Nature Methods</em> 6, no. 5 (April 12, 2009): 343–345. <a href="http://http://dx.doi.org/10.1038/nmeth.1318">doi:10.1038/nmeth.1318</a>.</p>
<p><a href="http://www.synbio.org.uk/dna-assembly/guidetogibsonassembly.html">http://www.synbio.org.uk/dna-assembly/guidetogibsonassembly.html</a></p>
